pymc3/variational/inference.py

```diff
@@ -408,6 +446,78 @@ def from_mean_field(cls, mean_field, gpu_compat=False):
 
     @classmethod
     def from_advi(cls, advi, gpu_compat=False):
+        """
+        Construct FullRankADVI from ADVI
+
+        Parameters
+        ----------
+        advi : ADVI
+
+        Flags
+        -----
+        gpu_compat : bool
+            use GPU compatible version or not
+
+        Returns
+        -------
+        FullRankADVI
+        """
         inference = cls.from_mean_field(advi.approx, gpu_compat)
         inference.hist = advi.hist
         return inference
+
+
+def approximate(n=10000, local_rv=None, method='advi', model=None, **kwargs):
```

Comment:
Another thought is that we already have `find_MAP`. That actually seems like a bad name which should probably just be `fit(method='map')`. So I'm fine with `fit()`. What do you think @ferrine?